src/main/java/io/confluent/connect/hdfs/HdfsSinkTask.java

```diff
@@ -93,6 +94,13 @@ public void start(Map<String, String> props) {
         hdfsWriter.stop();
       }
     }
+
+    log.info("HDFS connector does not commit consumer offsets to Kafka. Upon startup, HDFS "
+        + "Connector restores offsets from filenames in HDFS. In the absence of files in HDFS, "
+        + "the connector will attempt to find an offset for its consumer group in the "
+        + "'__consumer_offsets' topic. If no offsets are found in this topic, the consumer will "
```

Comment:
since plural is not accurate here anyways (`If no offsets are found` - the offset is one) should we make it shorter? `If none is found, the consumer will rely ... ` (or `If no offset is found`)

Reply:
aren't there offsets per partition?